test/e2e/set/scyllacluster/helpers.go

```diff
@@ -325,8 +337,8 @@ func getNodeName(sc *scyllav1.ScyllaCluster, idx int) string {
 	)
 }
 
-func restartRack(ctx context.Context, client appv1client.AppsV1Interface, r scyllav1.RackSpec, sc *scyllav1.ScyllaCluster) error {
-	sts, err := client.StatefulSets(sc.Namespace).Get(ctx, naming.StatefulSetNameForRack(r, sc), metav1.GetOptions{})
+func restartRack(ctx context.Context, appClient appv1client.AppsV1Interface, r scyllav1.RackSpec, sc *scyllav1.ScyllaCluster) error {
```

Comment:
nit: the API group name is `apps`, so it should be `appsClient` and `appsv1client`